aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,62 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 2023
```

Comment:
I think we should mention the Grid Fundamentals Part one tutorial in the April section.

Reply:
@jcfilben Can you point me to that ticket? Is that something that was completed in the previous sprint? If so, this What's New is only until mid April.